spec/sonar/cli_spec.rb

```diff
@@ -82,6 +82,29 @@
         end
       end
     end
+
+    describe 'sonar types command' do
+      it 'returns all sonar search types' do
+        output = run_command('types')
+        expect(output).to match(/Certificate to IPs/)
+      end
+    end
+
+    describe 'search_all command' do
```

Comment:
I think it would be a better experience if it was `sonar search all <term>` rather than `sonar search_all <term>`